Rubberduck.Parsing/VBA/DeclarationFinderManager.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Rubberduck.VBEditor.Application;
+using System;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public class DeclarationFinderManager : IDeclarationFinderManager 
```

Comment:
this is a glorified reference cache... wherever you feel you need to use this smells, because this whole class doesn't provide a tangible benefit. After all where the DeclarationFinder is used, it's usually defensively copied, and the DeclarationFinderManager won't help that. IMO this class is clutter.